editor/block-icon/index.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
This is on the boundary that I'd probably consider to be block specific (since it is defined as block API). I'd consider the inserter to be editor, but the block icon to be part of blocks.